ci/bamboo/build-osx.sh

```diff
@@ -9,6 +9,12 @@ export PYTHONPATH=${HOME}/Library/Python/2.7/lib/python/site-packages:${PYTHONPA
 # Install pip
 python ci/bamboo/get-pip.py --user --ignore-installed
 
+# Upgrade setuptools (for PEP-508 support used in extras_require); also wheel
+pip install --user --upgrade --ignore-installed setuptools wheel
+python -c 'import pip; print "pip version=", pip.__version__'
```

Comment:
Please remove these lines, or add them to `ci/bamboo/build-linux.sh` for consistency.